vmm/sandbox/src/stratovirt/devices/virtio_net.rs

```diff
@@ -62,9 +62,9 @@ pub struct VirtioNetDevice {
     pub(crate) fds: Vec<i32>,
     #[property(param = "netdev", predicate = "self.fds.len()<=0")]
     pub(crate) ifname: Option<String>,
-    #[property(param = "device", key = "bus")]
+    #[property(param = "device", key = "bus", predicate = "self.addr.len()>0")]
     pub(crate) pci_bus: Option<String>,
-    #[property(param = "device", key = "addr")]
```

Comment:
why 'addr' key is ignored, however 'bus' key is reserved?

Reply:
Because "addr" do not need to be renamed "addr", but "pci_bus" need to be renamed "bus".